src/html/header-nav-megamenu.html

```diff
@@ -0,0 +1,133 @@
+<ul class="usa-nav-primary usa-accordion">
+  <li>
+    <button class="usa-accordion-button usa-nav-link" aria-expanded="false" aria-controls="side-nav-1">
+      <span>Section title</span>
+    </button>
+    <ul class="usa-nav-submenu usa-megamenu" id="side-nav-1">
+      <div class="usa-grid-full">
+        <div class="usa-megamenu-col">
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+        </div>
+        <div class="usa-megamenu-col">
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+        </div>
+        <div class="usa-megamenu-col">
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+        </div>
+      </div>
+    </ul>
+  </li>
+  <li>
+    <button class="usa-accordion-button usa-nav-link" aria-expanded="false" aria-controls="sidenav-2">
+      <span>Section title</span>
+    </button>
+    <ul class="usa-nav-submenu usa-megamenu" id="sidenav-2">
+      <div class="usa-grid-full">
+        <div class="usa-megamenu-col">
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+        </div>
+        <div class="usa-megamenu-col">
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+        </div>
+        <div class="usa-megamenu-col">
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+          <li>
+            <a href="#">Subsection title</a>
+          </li>
+        </div>
+      </div>
+    </ul>
+  </li>
+  <li>
+    <a class="usa-nav-link" href="#">
+      <span>Section title</span>
+    </a>
+  </li>
+  <li>
+    <button class="usa-accordion-button usa-nav-link" aria-expanded="false" aria-controls="sidenav-3">
+      <span>Section title</span>
+    </button>
+    <ul class="usa-nav-submenu usa-megamenu" id="sidenav-3">
+      <div class="usa-grid-full">
```

Comment:
Could the usa-grid-full be mixed in to `usa-megamenu-col` to reduce one extra div of structure?